test/module/irohad/model/converters/json_query_factory_test.cpp

```diff
@@ -181,4 +186,55 @@ TEST(QuerySerializerTest, get_role_permissions){
   auto val = queryGenerator.generateGetRolePermissions();
   val->signature = generateSignature(42);
   runQueryTest(val);
-}
\ No newline at end of file
+}
+
+/**
+ * @given Random pager value
+ * @when serialize it then deserialize it
+ * @then Validate random pager value is equal to the expected value.
+ */
+struct TestPagerModel {
+  Pager pager{};
+};
+
+TEST(QuerySerializerTest, SerializePager) {
```

Comment:
Add `given, when, then`, pls.

Reply:
To test pager, I defined struct `TestPagerModel`. Test doc is above the struct.